src/combinatorics/set_family/compression/down.lean

```diff
@@ -0,0 +1,187 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import data.finset.card
+import data.fintype.basic
+
+/-!
+# Down-compressions
+
+This file defines down-compression. It is an operation on a set family that reduces the number of
```

Comment:
Let's not add this second sentence yet, because it hasn't been proved but also because we haven't defined shattering yet either.